db.json

```
@@ -29,5 +29,23 @@
       "value": "2",
       "date": "2025-05-08T15:42:46.335Z"
     }
+  ],
+  "my-cards": [
```

Sugestão, se fosse um endpoint real, provavelmente seria chamado dessa forma: 
```suggestion
  "cards": [
```